lib/game_choice.rb

```diff
@@ -0,0 +1,40 @@
+class Computer
+attr_accessor :c_choice
+  def initialize
+  @c_choice = ['rock','paper', 'scissors'].sample
+  end
+
+  def c_start
+  return @c_choice
+  end
+end
+
+class Game
+
+    def initialize
+      @computer = Computer.new
+    end
+
+  def start(choice)
+    computer =  @computer
+    if (choice == "rock") && (computer.c_choice == "rock")
+      return "Computer picked #{computer.c_choice}. It's a draw"
```

Comment:
Nice!

Reply:
It's great that you've got the logic here. There is an opportunity to refactor this making use of a hash. Can have a look at this solution as a guide: https://github.com/makersacademy/mastery-quiz-exemplars/blob/master/chapter6/quiz1/questions/question_2.rb